Sources/core/objects/Gun.cpp

```diff
@@ -0,0 +1,67 @@
+#include "Gun.h"
+
+Gun::Gun(size_t id) : Item(id)
+{
+    SetSprite("icons/guns.dmi");
+
+}
+
+bool Gun::Handle_Ammo()
+{
```

Comment:
Remove `_` in the function name. Also it is possible better to name it more clearly, for example `UseAmmo` (because "handle" is word with variety of meaning).